### ISSUE TITLE
Storefront: Wcag and accessibility review

### ISSUE THREAD

**Issue by Thuneer** [digdir/designsystem#239]
Quick review done
- [x] wave complains about a empty button on the menu. Screen reader do not pick up on it. 
- [x] give designsystem logo a better alt text. For example "Forside Designsystemet"
- [x] give decorative image on "God praksis" a better alt text or have none as it could be considered decorative.
- [x] add lang attribute to pages
- [x] some HTML validator issues. See example below (retest to see if its not a plugin or something on my computer doing it)
- [x] we should have a jump to main content button. It is most apparent on the "grunnleggende" page where you have to tab trough the whole left side menu to get to the content
- [x] missing H1 on "god praksis" and jumping over an heading level on "roadmap"
- [x] remove "f" as alt text on buttons with icon in the footer. (Slack, github and Figma)
- [x] something weird with the e-mail link in the footer when using screen reader. Se example below.
- [x] remove alt text for the slack logo in the "Bli med i Slack-kanalen" button.

Some extra mobile stuff nitpicky stuff
- [x] the menu on "grunnleggende" takes up a lot of space on small screens
- [x] when using the menu in the header, it stays open after you click a link. You cannot click outside it to close it, you have to use the "x" button.

![Skjermbilde 2023-06-01 kl  13 27 55](https://github.com/digdir/designsystem/assets/33222679/ce7e3411-0a23-4313-afc9-287f60ee901c)
![Skjermbilde 2023-06-01 kl  15 10 50](https://github.com/digdir/designsystem/assets/33222679/e86d19f5-80a1-4dfc-8a28-05558e94e8e9)
![Skjermbilde 2023-06-01 kl  15 10 14](https://github.com/digdir/designsystem/assets/33222679/1dd09813-ed26-45f5-8cca-b03a955536ba)

**Comment by mrosvik** [comment]
Also: 

![Image](https://github.com/digdir/designsystem/assets/1464915/7bddc340-f81e-4ee7-a960-6abe4ff518d4)

**Comment by mrosvik** [comment]
Jeg legger dette ut i en egen issue, slik at vi kan avslutte denne og merge inn https://github.com/digdir/designsystem/pull/504

**Comment by Febakke** [comment]
> Also:
> 
> ![Image](https://user-images.githubusercontent.com/1464915/245059321-7bddc340-f81e-4ee7-a960-6abe4ff518d4.png)

Jeg tror Øyvind la til språk. Hvilken side var det på?
